src/lib/components/Swap/Summary.fixture.tsx

```diff
@@ -18,24 +20,31 @@ const UNI = (function () {
 })()
 
 function Fixture() {
-  const [initialized, setInitialized] = useState(false)
   const setState = useUpdateAtom(swapAtom)
-  // eslint-disable-next-line react-hooks/exhaustive-deps
+
```

Comment:
nit: rm newline, and one-line the destructure from `useSwapInfo`:
` const { trade: { trade } } = useSwapInfo()`

Reply:
i think this is happenign because of different styling preferences - i just copied what some people were doing in app, but maybe we should all sync on those settings

Reply:
your styler usually allows it if you write it out that way.